packages/xod-client-browser/benchmark/puppeteerUtils.js

```diff
@@ -0,0 +1,24 @@
+const getBoundingClientRect = (page, handle) => page.evaluate((el) => {
```

Comment:
The same thing as before (`handle` -> `element` | `selector`)

Reply:
How about `elementHandle`? It does not sound like function and is consistent with official naming